mtags/src/main/scala/scala/meta/internal/mtags/KeywordWrapper.scala

```diff
@@ -63,14 +63,14 @@ object KeywordWrapper {
     "\u2190"
   )
 
+  // Hard keywords only
   val Scala3Keywords: Set[String] = Set(
     "abstract", "case", "catch", "class", "def", "do", "else", "enum", "export",
     "extends", "false", "final", "finally", "for", "given", "if", "implicit",
     "import", "lazy", "match", "new", "null", "object", "override", "package",
     "private", "protected", "return", "sealed", "super", "then", "throw",
     "trait", "true", "try", "type", "val", "var", "while", "with", "yield", ":",
-    "=", "<-", "=>", "<:", ":>", "#", "@", "=>>", "?=>", "extension", "inline",
-    "|", "*", "+", "-"
+    "=", "<-", "=>", "<:", ":>", "#", "@", "=>>", "?=>", "|", "*", "+", "-"
```

Comment:
We still need to add backticks for example in completions when doing:

```
def extension(a: String) = ???
extension("")
```
will cause an error.

Could we instead have a separate soft keywords set and not backtick them in specific scenarios? ?Using a flag for example here for the backticked methods here:

https://github.com/scalameta/metals/blob/main/mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala#L209

Reply:
Reverted this change, added tests